src/modules/automod/message.js

```diff
@@ -35,6 +35,8 @@ async function handleSwearing(client, message) {
             userDto.swearlevel = 0; // resets the swear level, if time ran out
         }
         userDto.swearlevel += 1;
+        userDto.cooldown.set('swearing', now + xpCooldown.swearing);
+        userDb.set(message.author.id, userDto);
```

Comment:
why exactly was this moved now?

Reply:
Because when calling penalize, it gets and sets a database value, where you then go and set your own value after overriting its changes like it never happened.